### PR TITLE
Add project People

### PR DESCRIPTION
**Pull request by ktquez**

### CODE REVIEW

File: src/project/people.md [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
 
-# Who make it happen.
+# Who does the project
+
+## Active team members
+
+<list-people role="team"></list-people>
 
-<br>
+## Package manteiners
```
Comment (marcus-herrmann) [inline src/project/people.md:20]: Typo: maintainers

**Review by marcus-herrmann — CHANGES_REQUESTED** [review]

File: src/project/people.md (hunk 1 above)
Comment (ktquez) [reply src/project/people.md:20]: Thanks @marcus-herrmann 
Done!

File: src/.vuepress/config/people.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    website: 'https://yamanoku.net',
+    role: 'team',
+    social: [
+      {
+        name: 'twitter',
+        username: 'yamanoku'
+      }
+    ],
+    avatar: '/img/people/yamanoku.png',
+    bio: 'FrontEnd Designer • Accessibility Adovocator • I want to be Internet.'
```
Comment (yamanoku) [inline src/.vuepress/config/people.js:28]:
```suggestion
    bio: 'FrontEnd Designer • Accessibility Adovocator'
```
This is it, please.

**Review by yamanoku — APPROVED** [review]
thanks ❤️

**Pull request merged**